examples/src/pages/Migration.tsx

```diff
@@ -485,6 +486,12 @@ export function Migration() {
             {
               const { treeIndex, point, model } = intersection;
               console.log(`Clicked node with treeIndex ${treeIndex} at`, point);
+              const overlayHtml = document.createElement('div');
+              overlayHtml.innerText = `Node ${treeIndex}`;
+              overlayHtml.style.cssText = 'background: white; position: absolute;';
+              overlayTool.add(overlayHtml, point);
+              console.log(overlayHtml);
```

Comment:
we are starting to get a lot of `console.log` clutter in the migration example. Kinda annoying when you are trying to debug something. Maybe we can remove some?

Reply:
Oops, removed